ecommerce/backend/API/utils/verify_email.py

```diff
@@ -0,0 +1,20 @@
+from django.template.loader import render_to_string
+from django.utils.encoding import force_bytes
+from django.core.mail import EmailMessage
+
+from API.utils.jwttoken import generate_access_token, generate_mail_token
+
+def email_send_verify(to_email, current_site, user):
+    mail_subject = 'Activate your account.'
+    message = render_to_string('acc_active_email.html', {
+                'user': user,
+                'domain': current_site.domain,
+                'url_path': 'email_verify',
+                'uid': generate_mail_token(user.pk),
+            })
```

Comment:
You probably need to add the domain ip as variable here for development/production environment.

Reply:
@bekir96 it relates you.